File: Watch Store App/WatchStoreApp-Frontend/watchstoreapp/src/Model/company.ts
export class company {
  constructor(
    public id: number = 0,
    public companyId: number = 0,
    public companyName: string = '',
    public companyImage: string = ''
  ) {}

  toString() {
    return (
      'Id is:- ' +
      this.id +
      'Company Id is:- ' +
      this.companyId +
      'Company Name is:- ' +
      this.companyName +
      'Company Image is:- ' +
      this.companyImage
    );
  }
}

File: Watch Store App/WatchStoreApp-Backend/watchstoreapp/src/main/java/com/mywatch/entity/Company.java
package com.mywatch.entity;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

@Entity
@Table(name="tbl_company")
public class Company {
	
	@Id
	private int id;
	@Column(nullable=false)
	private int companyId;
	@Column(nullable=false)
	private String companyName;
	@Column()
	private String companyImage;
	public int getId() {
		return id;
	}
	public void setId(int id) {
		this.id = id;
	}
	public int getCompanyId() {
		return companyId;
	}
	public void setCompanyId(int companyId) {
		this.companyId = companyId;
	}
	public String getCompanyName() {
		return companyName;
	}
	public void setCompanyName(String companyName) {
		this.companyName = companyName;
	}
	public String getCompanyImage() {
		return companyImage;
	}
	public void setCompanyImage(String companyImage) {
		this.companyImage = companyImage;
	}
	
	
	@Override
	public String toString() {
		return "Company [id=" + id + ", companyId=" + companyId + ", companyName=" + companyName + ", companyImage="
				+ companyImage + "]";
	}
	
}
